client/src/components/Slider.css

```diff
@@ -0,0 +1,35 @@
+.rec.rec-carousel-wrapper:focus,
+.rec.rec-carousel-wrapper .rec-carousel:focus,
+.rec.rec-carousel-wrapper .rec-slider-container:focus,
+.rec.rec-carousel-wrapper .rec-slider-container .rec-slider:focus,
+.rec.rec-carousel-wrapper .rec-slider-container .rec-slider div:focus {
+	outline: none;
+}
+
+.rec.rec-carousel-wrapper .rec-slider-container {
+	/* background: lightblue; */
```

Comment:
It's not a big issue but in general to keep code bases clean we never want to commit commented code

Reply:
Changed!